src/state/rootReducer.js

```diff
@@ -15,6 +15,7 @@ import fragments from 'state/fragments/reducer';
 import pageModels from 'state/page-models/reducer';
 import errors from 'state/errors/reducer';
 import dataTypes from 'state/data-types/reducer';
+import dataModelList from 'state/data-model-list/reducer';
```

Comment:
The state should be `data-model`, and should contain a `list` element: e.g.

**NO**
```
state
-- dataModelList
```

**YES**
```
state
-- dataModel
---- list
```